lib/connection.js

```diff
@@ -384,6 +384,8 @@ class Connection extends EventEmitter {
 
     this.server.startSession(this, application, authStrategy, credentials,
       this._onSessionCreated.bind(this));
+
+    this.emit('handshakeRequest', applicationName, authStrategy, credentials);
```

Comment:
@belochub also not sure about that, but I thought that users may decide for themselves whether they need to log them or just ignore this parameter. @metarhia/jstp-core?